service/DevDocs/SendAsyncTimeout.md

```diff
@@ -0,0 +1,36 @@
+# SendAsyncTimeout Requirements
+
+## Overview
+
+DeviceClient class is used by the device client callback delegate to prepare the callback results.
+
+## References
+
+##Exposed API
+
+```csharp
+
+sealed class AmqpServiceClient : ServiceClient
+{
+    ...
+    public async override Task SendAsync(string deviceId, Message message, TimeSpan timeout)
+}
+```
+
+### SendAsyncTimeout Thumbprint
```

Comment:
Not sure if "Thumbprint" is the right word here. Other reqs only mention the actual API name (in this case `SendAsync`).